questions.md

```diff
@@ -0,0 +1,5 @@
+ @ccannon94, 
+
+ If github pull request is sent in a forest, does it make a sound?
+ How many group members does it take to change a lightbulb and then correctly merge a pull request?
+ Wait, Can I accept my own pull request. 
```

Comment:
😂 😂 
No, bits on servers don't make sounds.
642 and one highly motivated hippopotamus.
Yes, yes you can. You can change those settings by protecting a branch in repository settings.